### PR TITLE
fix device map path

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: lib/cfa/grub2/device_map.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ module Grub2
     #   device.
     class DeviceMap < BaseModel
       PARSER = AugeasParser.new("device_map.lns")
-      PATH = "/etc/grub2/device.map".freeze
+      PATH = "/boot/grub2/device.map".freeze
```
Comment (mvidner) [inline lib/cfa/grub2/device_map.rb]: Cough, line 8.

**Comment by lslezak** [comment]
LGTM

(BTW Do we need a `bsc` number for this?)

**Comment by jreidinger** [comment]
@mvidner thanks for catch
@lslezak no, as older change done today already contain FATE number

**Comment by lslezak** [comment]
LGTM

**Pull request merged**